gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Thu Sep 15 13:57:35 ART 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-bin.zip
```

Comment:
When using the `wrapper` task to update the Gradle wrapper version, please consider doing

    ./gradlew wrapper  --gradle-version 4.10.2 --distribution-type all

The _default_ for Android projects is the `all` variant, which provides additional documentation and sources helpful when working on Gradle related stuff. Since it's the default it's also much more likely to be already downloaded in a Gradle cache. It is a _one-time, machine-wide_ download.

Reply:
Thanks! I'll update it